app/views/films/index.html.erb

```diff
@@ -5,7 +5,7 @@
 
 <h1>Films Index</h1>
 <% @films.nominated_only.each do |film| %>
-  <h3><%= film.name %></h3>
+  <h3 id="<%= film.id %>"><%= film.name %> - <%= link_to 'Edit Film', "/films/#{film.id}/edit" %></h3>
```

Comment:
Nice, you added it to both pages too. Re-reading the user story it said we only needed to add an edit link to one of the pages on ONE of our projects. All well, thoroughness is preferred in my opinion. :)